docs(explainer): Update the explainer to include digital credential issuance (#364)
w3c-fedid/digital-credentials: explainer.md

# Digital Credentials API

> Government recognized digital credentials (e.g., driver’s licenses, passports) and other digital attestations are being increasingly utilized on the web. We propose a deliberate and flexible API to enable browsers to mediate both the *presentation* and *issuance* of these credentials, aiming to continuously improve the balance of benefits and risks for users and the entire online community.

Reviewer: ```suggestion
Government-recognized digital credentials (e.g., driver’s licenses, passports) and other digital attestations are increasingly being used on the web. We propose a deliberate and flexible API to enable browsers to mediate both the *presentation* and *issuance* of these credentials, aiming to continuously improve the balance of benefits and risks for users and the entire online community.
```

# Why?

Government-recognized documents play a big and constructive role in society (e.g., drivers licenses, passports, etc.). Increasingly, with the movement of government and financial services online, and regulation (e.g. [eIDAS](https://en.wikipedia.org/wiki/EIDAS) and various [age verification regulations](https://en.wikipedia.org/wiki/Age_verification_system)), these paper-based documents are gaining digital counterparts

> Along with all of their potential from the physical world, the issuance and presentation of government-recognized digital credentials also brings their associated risks of abuse, such as the potential for an increase in [surveillance](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#no-phoning-home), [censorship](https://github.com/w3cping/credential-considerations/blob/main/risks.md#censorship-and-reduction-in-access-to-free-information), [discrimination](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#free-expression), and intrusion to the online world.

Reviewer 2: @npdoty's link fixes 

```suggestion
Along with all of their potential from the physical world, the issuance and presentation@ of government-recognized digital credentials also brings their associated risks of abuse, such as the potential for an increase in [surveillance](https://github.com/w3c/credential-considerations/blob/main/credentials-considerations.md#invasions-of-privacy), [censorship](https://github.com/w3c/credential-considerations/blob/main/credentials-considerations.md#restrictions-of-free-expression), [discrimination](https://github.com/w3c/credential-considerations/blob/main/credentials-considerations.md#restrictions-of-free-expression), and intrusion to the online world.
```

> Along with all of their potential from the physical world, the issuance and presentation of government recognized  digital credentials also brings their associated risks of abuse, such as the potential for an increase in [surveillance](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#no-phoning-home), [censorship](https://github.com/w3cping/credential-considerations/blob/main/risks.md#censorship-and-reduction-in-access-to-free-information), [discrimination](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#free-expression), and intrusion to the online world.

Reviewer: ```suggestion
Along with all of their potential from the physical world, the issuance and presentation of government-recognized digital credentials also brings their associated risks of abuse, such as the potential for an increase in [surveillance](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#no-phoning-home), [censorship](https://github.com/w3cping/credential-considerations/blob/main/risks.md#censorship-and-reduction-in-access-to-free-information), [discrimination](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#free-expression), and intrusion to the online world.
```

> Along with all of their potential from the physical world, the issuance and presentation of government recognized  digital credentials also brings their associated risks of abuse, such as the potential for an increase in [surveillance](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#no-phoning-home), [censorship](https://github.com/w3cping/credential-considerations/blob/main/risks.md#censorship-and-reduction-in-access-to-free-information), [discrimination](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#free-expression), and intrusion to the online world.

Reviewer 2: Added Nick's links.

> Along with all of their potential from the physical world, the issuance and presentation of government recognized  digital credentials also brings their associated risks of abuse, such as the potential for an increase in [surveillance](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#no-phoning-home), [censorship](https://github.com/w3cping/credential-considerations/blob/main/risks.md#censorship-and-reduction-in-access-to-free-information), [discrimination](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#free-expression), and intrusion to the online world.

Author: Done!

In the pursuit of establishing a standard for web-based identity sharing, it is paramount that we not only strive for ease of use but also exercise utmost caution to uphold high standards of security and privacy. This is essential to safeguard users from potential identity theft and ensure they are fully informed about the implications of online identity sharing before providing their consent. We firmly believe that relying on a browser API and the mobile platform is the most reliable approach for identity digital credentials to be issued and presented securely online.

# What?

> First we must acknowledge, architecturally, that this is a rapidly evolving space with a large number of moving parts at various different levels of maturity. Users, regulations (e.g. eIDAS), wallets, verifiers, issuers, formats (e.g. [ISO mDocs](https://www.iso.org/obp/ui/en/#iso:std:iso-iec:18013:-5:ed-1:v1:en) and [W3C VCs](https://verifiablecredentials.dev/), just to name a few) and protocols (e.g. [OpenID4VP](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html), [OpenID4VCI](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html), ISO REST’s API, [VC API](https://github.com/w3c-ccg/vc-api)), operating systems and browsers (e.g. evolving mitigation strategies, for [example](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit#heading=h.8gq5f7p3it8q)) are all still in various stages of formation and need the space and autonomy to evolve efficiently.

Reviewer: ```suggestion
First we must acknowledge, architecturally, that this is a rapidly evolving space with a large number of moving parts at various different levels of maturity. Users, regulations (e.g., eIDAS), wallets, verifiers, issuers, formats (e.g., [ISO mDocs](https://www.iso.org/obp/ui/en/#iso:std:iso-iec:18013:-5:ed-1:v1:en) and [W3C VCs](https://verifiablecredentials.dev/), just to name a few) and protocols (e.g., [OpenID4VP](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html), [OpenID4VCI](https://openid.github.io/OpenID4VCI/openid-4-verifiable-credential-issuance-wg-draft.html), ISO REST API, [VC API](https://github.com/w3c-ccg/vc-api)), operating systems, and browsers (e.g., evolving mitigation strategies, for [example](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit#heading=h.8gq5f7p3it8q)) are all still in various stages of formation and need the space and autonomy to evolve efficiently.
```

> First we must acknowledge, architecturally, that this is a rapidly evolving space with a large number of moving parts at various different levels of maturity. Users, regulations (e.g. eIDAS), wallets, verifiers, issuers, formats (e.g. [ISO mDocs](https://www.iso.org/obp/ui/en/#iso:std:iso-iec:18013:-5:ed-1:v1:en) and [W3C VCs](https://verifiablecredentials.dev/), just to name a few) and protocols (e.g. [OpenID4VP](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html), [OpenID4CI](https://openid.github.io/OpenID4CI/openid-4-verifiable-credential-issuance-wg-draft.html), ISO REST’s API, [VC API](https://github.com/w3c-ccg/vc-api)), operating systems and browsers (e.g. evolving mitigation strategies, for [example](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit#heading=h.8gq5f7p3it8q)) are all still in various stages of formation and need the space and autonomy to evolve efficiently.

Reviewer 3: ```suggestion
First we must acknowledge, architecturally, that this is a rapidly evolving space with a large number of moving parts at various different levels of maturity. Users, regulations (e.g. eIDAS), wallets, verifiers, issuers, formats (e.g. [ISO mDocs](https://www.iso.org/obp/ui/en/#iso:std:iso-iec:18013:-5:ed-1:v1:en) and [W3C VCs](https://verifiablecredentials.dev/), just to name a few) and protocols (e.g. [OpenID4VP](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html), [OpenID4VCI](https://openid.github.io/OpenID4CI/openid-4-verifiable-credential-issuance-wg-draft.html), ISO REST’s API, [VC API](https://github.com/w3c-ccg/vc-api)), operating systems and browsers (e.g. evolving mitigation strategies, for [example](https://docs.google.com/document/d/1L68tmNXCQXucsCV8eS8CBd_F9FZ6TNwKNOaFkA8RfwI/edit#heading=h.8gq5f7p3it8q)) are all still in various stages of formation and need the space and autonomy to evolve efficiently.
```

- Websites cannot access any information without user interaction. This API ensures that sites cannot silently query for or issue digital credentials or communicate with wallet providers without the user's active participation and confirmation of each action.

At its core, the API is designed for a website ("verifier") to [transparently](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#in-context-explanations) request the [selective disclosure](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#selective-disclosure) of attributes from (issued) digital credentials that were provisioned - ahead of time - to wallets ("holders"), and for a website ("issuer") to [transparently](https://github.com/w3cping/credential-considerations/blob/main/credentials-considerations.md#in-context-explanations) initiate the issuance of new credentials to a user's wallet, in a manner that is seamlessly compatible with existing architectural choices (such as [OpenID4VP integration](https://github.com/openid/OpenID4VP/issues/125)).

> For example of how to use the API to [request](https://w3c-fedid.github.io/digital-credentials/#example-requesting-a-digital-credential) or [issue](https://w3c-fedid.github.io/digital-credentials/#example-issuing-a-digital-credential) a digital credential and a more detailed technical description, please refer to the [specification draft](https://www.w3.org/TR/digital-credentials/).

Reviewer 3: ```suggestion
For example of how to use the API to [request](https://www.w3.org/TR/digital-credentials/#example-requesting-a-digital-credential) or [issue](https://www.w3.org/TR/digital-credentials/#example-issuing-a-digital-credential) a digital credential and a more detailed technical description, please refer to the [specification draft](https://www.w3.org/TR/digital-credentials/).
```
